bin/strata-cli/src/constants.rs

```diff
@@ -38,9 +38,8 @@ pub const DEFAULT_ESPLORA: &str = "https://esploraapi.devnet-annapurna.stratabtc
 pub const DEFAULT_L2_HTTP_ENDPOINT: &str = "https://stratareth.devnet-annapurna.stratabtc.org";
 pub const DEFAULT_FAUCET_ENDPOINT: &str = "https://faucet.devnet-annapurna.stratabtc.org";
 pub const DEFAULT_MEMPOOL_ENDPOINT: &str = "https://mempool.devnet-annapurna.stratabtc.org";
-// FIXME: CHANGE BELOW!!!
 pub const BRIDGE_MUSIG2_PUBKEY: &str =
-    "fbd79b6b8b7fe11bad25ae89a7415221c030978de448775729c3f0a903819dd0";
+    "51acd1574ef6b1fb98ed000db6e31a7cb8d3dba302fa22e2c99a03f4cbfc2438";
```

Comment:
Is this the official one?

Reply:
@Zk2u  Please use this key:
14ced579c6a92533fa68ccc16da93b41073993cfc6cc982320645d8e9a63ee65